accelerator/config.c

```diff
@@ -70,7 +83,19 @@ status_t cli_config_set(ta_config_t* const info, iota_config_t* const iconf, ta_
       iconf->seed = value;
       break;
     case CACHE:
-      cache->cache_state = !(strncmp(value, "T", 1));
+      cache->cache_state = (toupper(value[0]) == 'T');
+      break;
+
+    // Verbose configuration
+    case VERBOSE:
+      verbose_mode = (toupper(value[0]) == 'T');
+      break;
+
+    // File configuration
+    case CONF_CLI:
+      strncpy(conf_file, value, arg_len);
+      conf_file[arg_len] = '\0';
+      break;
```

Comment:
Actually we don't need to do anything `CONF_CLI`, right?
Since the configuration file setting is already set.

Reply:
If `CONF_CLI` matched, we have to save file path.
In general, `--config` will only appear at most once in CLI. Therefore, 
```
    case CONF_CLI:
      strncpy(conf_file, value, arg_len);
      conf_file[arg_len] = '\0';
      break;
```
will only be executed at first time.